### PR TITLE
[Testing] Impl sys_usbd_get_device_location

### PR DESCRIPTION
**Pull request by RipleyTom**
Also removed some redundant code that resulted in warnings.
Fixes https://github.com/RPCS3/rpcs3/issues/11174.

### CODE REVIEW

File: rpcs3/Emu/Cell/lv2/sys_usbd.cpp [hunk 1]
```diff
@@ -931,11 +928,21 @@ error_code sys_usbd_get_isochronous_transfer_status(ppu_thread& ppu, u32 handle,
 	return CELL_OK;
 }
 
-error_code sys_usbd_get_device_location(ppu_thread& ppu)
+error_code sys_usbd_get_device_location(ppu_thread& ppu, u32 handle, u32 device_handle, vm::ptr<u8> location)
 {
 	ppu.state += cpu_flag::wait;
 
-	sys_usbd.todo("sys_usbd_get_device_location()");
+	sys_usbd.notice("sys_usbd_get_device_location(handle=0x%x, device_handle=0x%x, location=*0x%x)", handle, device_handle, location);
```
Comment (Megamouse) [inline rpcs3/Emu/Cell/lv2/sys_usbd.cpp:935]: Missing param check for location

Comment (RipleyTom) [reply rpcs3/Emu/Cell/lv2/sys_usbd.cpp:935]: It's a syscall, there is generally no parameter check.
There is one in cellUsbd function which is lled.

**Comment by shockdude** [comment]
A DJH2 player gave me permission to share his log (with sys_usbd: Trace set). He says that even on the latest test build, 2 tables plugged in results in the game not bootig past the PPU screen.
[RPCS3.log](https://github.com/RPCS3/rpcs3/files/7592000/RPCS3.log)

**Comment by Crafty-The-Fox** [comment]
> A DJH2 player gave me permission to share his log (with sys_usbd: Trace set). He says that even on the latest test build, 2 tables plugged in results in the game not bootig past the PPU screen. [RPCS3.log](https://github.com/RPCS3/rpcs3/files/7592000/RPCS3.log)



Tried with the even newer build. Still blackscreens, and i did  log sys_usbd: Trace, heres the log
[RPCS3.log](https://github.com/RPCS3/rpcs3/files/7598878/RPCS3.log)
.

**Comment by Crafty-The-Fox** [comment]
Tried the even newer build. Game boots now, but if you press any button or anything on the turntable, game locks up
[RPCS3.log](https://github.com/RPCS3/rpcs3/files/7598906/RPCS3.log)
.

**Comment by RipleyTom** [comment]
> Tried the even newer build. Game boots now, but if you press any button or anything on the turntable, game locks up. [RPCS3.log](https://github.com/RPCS3/rpcs3/files/7598906/RPCS3.log)
> 
> 1 Dongle with 1 turntable still works just fine.

Looked at the log and I'm not sure what's up with it. It clearly communicates fine with the 2 turntables.

**Comment by Crafty-The-Fox** [comment]
This somehow breaks support for extracted DISC0.IMG, witch works completely fine on a PS3. i am not sure what DJ Hero dosent like about this, but il be writing up a github report soon.

**Pull request merged**